test/system/src/test/java/io/pravega/test/system/PravegaTest.java

```diff
@@ -155,7 +155,7 @@ public void simpleTest() {
                 fail("Reinitialization Exception is not expected");
             }
             // try reading until all the written events are read, else the test will timeout.
-        } while ((event.getEvent() != null || event.isCheckpoint()) && readCount < NUM_EVENTS);
+        } while (readCount < NUM_EVENTS);
```

Comment:
This is weakening the verification, why? We don't really expect read timeouts, so perhaps the right way of fixing this is to increase the timeout value.

Reply:
done.